megach.py

```diff
@@ -589,47 +587,21 @@ def RPOST(url, data = None, headers = None):
         except Exception as e:
             raise  # TODO no controlada
 
-
-def User(name: str, **kwargs):
-    """
-    Un def que representa a la clase _User. Si el usuario ya existe, mejor regresa ese
-    y en caso de que no, lo crea y lo agrega a la lista
-    @param name: Nombre del usuario
-    @param kwargs: Datos que contendrá el usuario
-    @return: Usuario nuevo o encontrado en la lista
-    """
-    if name is None:
-        name = ""
-    user = _users.get(name.lower())
-
-    if not user:
-        user = _User(name = name, **kwargs)
-        _users[name.lower()] = user
-    return user
-
-
-class _User:
+class User:
     """
     Clase que representa a un usuario de chatango
     Iniciarlo sin el guion bajo para evitar inconvenientes
     """
+    _users = []
```

Comment:
Found it, this needs to be a dict.

Reply:
Already fixed in https://github.com/TheClonerx/megach/commit/f41d30868e2678c508ad66a2221a29ad046f303f